auth0_flutter_platform_interface/lib/src/web-auth/safari_view_controller.dart

```diff
@@ -0,0 +1,32 @@
+/// Presentation styles for when using SFSafariViewController on iOS.
+/// For the full description of what each option does, please see the
+/// [UIModelPresentationStyle docs](https://developer.apple.com/documentation/uikit/uimodalpresentationstyle).
+enum SafariViewControllerPresentationStyle {
+  automatic(-2),
+  none(-1),
+  fullScreen(0),
+  pageSheet(1),
+  formSheet(2),
+  currentContext(3),
+  custom(4),
+  overFullScreen(5),
+  overCurrentContext(6),
+  popover(7);
+
+  const SafariViewControllerPresentationStyle(this.value);
+  final int value;
+}
+
+/// Configuration for using `SFSafariViewController` on iOS.
+class SafariViewController {
+  /// The presentation style used when opening the browser window. Defaults to `fullScreen`.
+  final SafariViewControllerPresentationStyle? presentationStyle;
+
+  const SafariViewController({this.presentationStyle});
+
+  Map<String, dynamic> toMap() => {
+        ...presentationStyle != null
+            ? {'presentationStyle': presentationStyle?.value}
+            : <String, dynamic>{}
```

Comment:
Can we get this line covered by a test?

Reply:
Also, would it work the same if it always returned a map with a `presentationStyle` key, but the value being `null` if not present?

Reply:
I think it would work the same, on the native side it would default to `fullScreen`.

Reply:
I've simplified it to always send the key + `null` if it's not set 👍🏻